gemrb/core/GameData.h

```diff
@@ -58,6 +60,11 @@ struct Table {
 	unsigned int refcount;
 };
 
+struct ieResRefHash
```

Comment:
let's move this to `ResRef`

Reply:
Done, and I also found a few more NULLs. Squashed and pushed the update.

Reply:
thanks